content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -1,206 +1,261 @@
-    .WithTracing(tracerProviderBuilder =>
-        tracerProviderBuilder
-            .AddSource(DiagnosticsConfig.ActivitySource.Name)
-            .ConfigureResource(resource => resource
-                .AddService(DiagnosticsConfig.ServiceName))
-            .AddAspNetCoreInstrumentation()
-            .AddConsoleExporter());
+app.MapGet("/rolldice/{player?}", HandleRollDice);
 
-// ... other setup
+app.Run();
+return;
 
-public static class DiagnosticsConfig
+int RollDice()
 {
-    public const string ServiceName = "MyService";
-    public static ActivitySource ActivitySource = new ActivitySource(ServiceName);
+    return random.Next(1, 7);
 }
-```
-
-At this stage, you should be able to run your site, and see a Console output
-similar to this:
 
-Note: an `Activity` in .NET is analogous to a Span in OpenTelemetry terminology
+string HandleRollDice(string? player)
+{
+    var result = RollDice();
+
+    if (string.IsNullOrEmpty(player))
+    {
+        logger.LogInformation("Anonymous player is rolling the dice: {result}", result);
+    }
+    else
+    {
+        logger.LogInformation("{player} is rolling the dice: {result}", player, result);
+    }
+
+    return result.ToString(CultureInfo.InvariantCulture);
+}
+```
 
-<details>
-<summary>View example output</summary>
+In the `Properties` subdirectory, replace the content of `launchSettings.json`
+with the following:
 
-```properties
-Activity.TraceId:            54d084eba205a7a39398df4642be8f4a
-Activity.SpanId:             aca5e39a86a17d59
-Activity.TraceFlags:         Recorded
-Activity.ActivitySourceName: Microsoft.AspNetCore
-Activity.DisplayName:        /
-Activity.Kind:               Server
-Activity.StartTime:          2023-02-21T12:19:28.2499974Z
-Activity.Duration:           00:00:00.3106744
-Activity.Tags:
-    net.host.name: localhost
-    net.host.port: 5123
-    http.method: GET
-    http.scheme: http
-    http.target: /
-    http.url: http://localhost:5123/
-    http.flavor: 1.1
-    http.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.50
-    http.status_code: 200
-Resource associated with Activity:
-    service.name: MyService
-    service.instance.id: 2c7ca153-e460-4643-b550-7c08487a4c0c
+```json
+{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:8080",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
+}
 ```
 
-</details>
+Build and run the application with the following command, then open
+<http://localhost:8080/rolldice> in your web browser to ensure it is working.
 
-### Manual Instrumentation
+```sh
+dotnet build
+dotnet run
+```
 
-Next, add [tracing](/docs/concepts/signals/traces/) via the `System.Diagnostics`
-API.
+## Instrumentation
```

Comment:
For this section, I'd also like to see a Windows tab for the install link and command. Similarly, the env vars should have an appropriate Windows section. I think a ton of .NET developers will try stuff out on a Windows machine but then deploy to Linux, and the current link won't work unless they're on WSL.

Reply:
Good point. To be honest, I was using github shell on Windows to write and tests this document so I have missed that PowerShell also covered. It should work on all systems if you have linux-like shell.

Reply:
Fixed. PowerShell scripts tested on newly created Windows 11 Azure. The only manual changes - installation .NET SDK 6.